<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScatterplotViewer</class>
 <widget class="QMainWindow" name="ScatterplotViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ProjectViewer/icons.qrc">
    <normaloff>:/icons/images/logo_fs_64x64.png</normaloff>:/icons/images/logo_fs_64x64.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="ScatterplotView" name="scatterplotView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>S&amp;cale</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionZoom_Fit_Window"/>
   </widget>
   <widget class="QMenu" name="menu_Display">
    <property name="title">
     <string>D&amp;isplay</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionDisplay_Options"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_Data">
    <property name="title">
     <string>&amp;Data</string>
    </property>
    <widget class="QMenu" name="menuHistogram">
     <property name="title">
      <string>&amp;Histogram</string>
     </property>
     <addaction name="action_HistogramXAxis"/>
     <addaction name="action_HistogramYAxis"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_Items"/>
    <addaction name="menuHistogram"/>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Data"/>
   <addaction name="menu_Display"/>
   <addaction name="menu_Window"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="mouseToolBar">
   <property name="windowTitle">
    <string>Mouse Mode Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="zoomToolBar">
   <property name="windowTitle">
    <string>zoomToolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>General Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelect_Items"/>
   <addaction name="actionDisplay_Options"/>
   <addaction name="separator"/>
   <addaction name="action_HistogramXAxis"/>
   <addaction name="action_HistogramYAxis"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="../ProjectViewer/icons.qrc">
     <normaloff>:/icons/Zoom-Plus-icon.png</normaloff>:/icons/Zoom-Plus-icon.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="../ProjectViewer/icons.qrc">
     <normaloff>:/icons/Zoom-Minus-icon.png</normaloff>:/icons/Zoom-Minus-icon.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionZoom_Fit_Window">
   <property name="text">
    <string>Zoom &amp;Fit Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionDisplay_Options">
   <property name="icon">
    <iconset resource="../ProjectViewer/icons.qrc">
     <normaloff>:/icons/images/display-options-icon-32x32.png</normaloff>:/icons/images/display-options-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>Display &amp;Options</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_HistogramXAxis">
   <property name="icon">
    <iconset resource="../ProjectViewer/icons.qrc">
     <normaloff>:/icons/images/histogram-x-axis-icon-32x32.png</normaloff>:/icons/images/histogram-x-axis-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>&amp;x-axis</string>
   </property>
  </action>
  <action name="action_HistogramYAxis">
   <property name="icon">
    <iconset resource="../ProjectViewer/icons.qrc">
     <normaloff>:/icons/images/histogram-y-axis-icon-32x32.png</normaloff>:/icons/images/histogram-y-axis-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>&amp;y-axis</string>
   </property>
  </action>
  <action name="actionSelect_Items">
   <property name="icon">
    <iconset resource="../ProjectViewer/icons.qrc">
     <normaloff>:/icons/images/select-range2-icon-32x32.png</normaloff>:/icons/images/select-range2-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>Select &amp;Items</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScatterplotView</class>
   <extends>QGraphicsView</extends>
   <header>scatterplotview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../ProjectViewer/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionE_xit</sender>
   <signal>triggered()</signal>
   <receiver>ScatterplotViewer</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
